Source/Scene/GlobeDepth.js

```diff
@@ -56,194 +50,25 @@ function GlobeDepth() {
 Object.defineProperties(GlobeDepth.prototype, {
   framebuffer: {
     get: function () {
-      return this._globeColorFramebuffer;
-    },
-  },
-  primitiveFramebuffer: {
-    get: function () {
-      return this._primitiveColorFramebuffer;
+      return this._colorFramebuffer.framebuffer;
     },
   },
 });
 
-function destroyTextures(globeDepth) {
-  globeDepth._globeColorTexture =
-    globeDepth._globeColorTexture &&
-    !globeDepth._globeColorTexture.isDestroyed() &&
-    globeDepth._globeColorTexture.destroy();
-  globeDepth._depthStencilTexture =
-    globeDepth._depthStencilTexture &&
-    !globeDepth._depthStencilTexture.isDestroyed() &&
-    globeDepth._depthStencilTexture.destroy();
-  globeDepth._globeDepthTexture =
-    globeDepth._globeDepthTexture &&
-    !globeDepth._globeDepthTexture.isDestroyed() &&
-    globeDepth._globeDepthTexture.destroy();
-}
-
 function destroyFramebuffers(globeDepth) {
-  globeDepth._globeColorFramebuffer =
-    globeDepth._globeColorFramebuffer &&
-    !globeDepth._globeColorFramebuffer.isDestroyed() &&
-    globeDepth._globeColorFramebuffer.destroy();
-  globeDepth._copyDepthFramebuffer =
-    globeDepth._copyDepthFramebuffer &&
-    !globeDepth._copyDepthFramebuffer.isDestroyed() &&
-    globeDepth._copyDepthFramebuffer.destroy();
+  globeDepth._colorFramebuffer.destroyResources();
```

Comment:
If `FramebufferManager.destroyResources` will no longer call `destroyFramebuffer`, this code will need to be updated.